.github/workflows/tests.yml

```diff
@@ -8,6 +8,17 @@ on:
   - pull_request
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - uses: pre-commit/action@v3.0.0
+      - uses: pre-commit-ci/lite-action@v1.0.1
+        if: always()
```

Comment:
Alternative: use https://pre-commit.ci/ instead. However it needs a maintainer to authorise it